libcloudforensics/providers/aws/internal/s3.py

```diff
@@ -118,3 +128,41 @@ def Put(self, s3_path: str, filepath: str) -> None:
           'Could not upload file {0:s}: {1:s}'.format(
               filepath, str(exception)),
           __name__) from exception
+
+  def GCSToS3(self,
```

Comment:
Interesting to have placed this in `aws.internal.s3`. Wouldn't this be better placed in another "cross-cloud" module? We can leave this for a future PR, but please open an issue and add a TODO if that's the case.

Reply:
I put it in S3 since all the "write" operations are on S3 : CreateBucket and Put.
The `get` seemed more incidental to me.
Interestingly, S3ToGCS is a pure GCP operation and thus was planning to be in the `gcp` module. 
A cross-cloud one seems ok though.

Reply:
Sounds good. Can we open an issue to get this migrated there?

Reply:
https://github.com/google/cloud-forensics-utils/issues/297